psi4/src/export_functional.cc

```diff
@@ -261,8 +261,13 @@ void export_functional(py::module &m) {
         .def("blocks", &MolecularGrid::blocks, "Returns a list of blocks.");
 
     py::class_<DFTGrid, std::shared_ptr<DFTGrid>, MolecularGrid>(m, "DFTGrid", "docstring")
-        .def_static("build", [](std::shared_ptr<Molecule> &mol, std::shared_ptr<BasisSet> &basis) {
-            return DFTGrid(mol, basis, Process::environment.options);
+        .def_static("build",
+                    [](std::shared_ptr<Molecule> &mol, std::shared_ptr<BasisSet> &basis) {
+                        return std::shared_ptr<DFTGrid>(new DFTGrid(mol, basis, Process::environment.options));
```

Comment:
I'm sure everyone realizes that I'm a one-trick pony wrt C++ style, but can this be:
`return std::make_shared<DFTGrid>(mol, basis, Process::environment.options);`